scripts_future_API/tm_stokes_mpi_wip.jl

```diff
@@ -59,7 +60,7 @@ function main(; do_visu=false, do_save=false, do_h5_save=false)
 
     grid_l = local_grid(grid_g, topo)
 
-    global_logger(FastIce.Logging.MPILogger(0, comm, global_logger()))
+    # global_logger(FastIce.Logging.MPILogger(0, comm, global_logger()))
```

Comment:
@utkinis this currently fails

Reply:
This could be addressed in a separate PR